content/en/continuous_delivery/deployments/argocd.md

```diff
@@ -75,8 +75,11 @@ The following resources have been added:
 3. The `cd-visibility-trigger` defines when to send the notification, and it references the `cd-visibility-template`.
 
 <div class="alert alert-warning">
-The call to populate the <code>commit_metadata</code> field is not required. The field is used to enrich the payload with Git information.
-If your Argo CD application source is not a defined commit SHA (for example, if you are using Helm repositories), adjust the body by removing that line and the comma in the previous line.
+The <code>commit_metadata</code> field is optional and can be used to enrich the deployment with Git information. It should be removed (together with the comma in the previous line) in the following cases:
+<ul>
+<li>You are already syncing your repository information to Datadog (see <a href="/continuous_delivery/features/code_changes_detection#synchronize-repository-metadata-to-datadog">synchronize your repository metadata to Datadog</a>).</li>
+<li>Your Argo CD application source does not have a defined commit SHA (for example, if you are using Helm repositories).</li>
+</ul>
```

Comment:
Alerts aren't really meant to contain lists or other detailed content, so the result looks awkward (see screenshot). Would it make sense to just use regular copy under a dedicated header or something like that? Let me know if you want help with it!

<img width="784" alt="Screenshot 2025-03-19 at 1 05 57 PM" src="https://github.com/user-attachments/assets/1ac2df58-8332-49c7-821f-11a65d985cf8" />

Reply:
Good spot, thanks! I have removed the alert format :)